src/crawlee/playwright_crawler/types.py

```diff
@@ -6,12 +6,19 @@
 from crawlee.basic_crawler.types import BasicCrawlingContext, EnqueueLinksFunction
 
 if TYPE_CHECKING:
-    from playwright.async_api import Page
+    from playwright.async_api import Page, Response
 
 
 @dataclass(frozen=True)
 class PlaywrightCrawlingContext(BasicCrawlingContext):
-    """Crawling context used by PlaywrightSoupCrawler."""
+    """Crawling context used by PlaywrightSoupCrawler.
+
+    Args:
+        page: The Playwright `Page` object.
+        response: The Playwright `Response` object.
+        enqueue_links: The `PlaywrightCrawler` implementation of the `EnqueueLinksFunction` function.
```

Comment:
Is this correctly rendered in the documentation?

Reply:
Unfortunately, args in API docs currently do not work - https://github.com/apify/crawlee-python/issues/324.